resources/asciidoctor/lib/edit_me/extension.rb

```diff
@@ -12,32 +12,56 @@ class EditMe < TreeProcessorScaffold
 
   def process(document)
     logger.error("sourcemap is required") unless document.sourcemap
-    super if document.attributes['edit_url']
+    edit_urls_string = document.attributes['edit_urls']
+    return unless edit_urls_string
+
+    edit_urls = []
+    CSV.parse edit_urls_string do |toplevel, url|
+      unless toplevel
+        logger.error message_with_context "invalid edit_urls, no toplevel"
+        next
+      end
+      unless url
+        logger.error message_with_context "invalid edit_urls, no url"
+        next
+      end
+      url = url[0..-2] if url.end_with? '/'
```

Comment:
Since you are mutating `url` anyway, you could:

```
url.chomp!('/')
```

Not a change request, just a "fun fact".